app/src/main/res/values/strings.xml

```diff
@@ -16,6 +16,9 @@
     <string name="undo">UNDO</string>
     <string name="delete">DELETE</string>
     <string name="save">SAVE</string>
```

Comment:
I know these aren't modified by your PR, but would you mind fixing the capitalisation (e.g. _Undo_ instead of _UNDO_)? I don't know why are they uppercase in the first place, but makes the "SAVE" button look weird when renaming a note.